tox.ini

```diff
@@ -28,6 +28,7 @@ passenv =
     dev: H_CLIENT_SECRET
     dev: H_JWT_CLIENT_ID
     dev: H_JWT_CLIENT_SECRET
+    dev: MAILCHIMP_API_KEY
```

Comment:
We'll get one of this in devdata? I wonder what the right approach is. 

You shouldn't have real emails in your local DB but I'm a bit paranoid about this, maybe we could force the flag that doesn't use the real emails with an envvar?

Reply:
Hard to say: I'm not sure if we want dev environments doing things with our actual Mailchimp account.

In theory I could create a devdata Mailchimp subaccount and devdata API key to make it easy for devs to test against the real Mailchimp. And then maybe I'd just add a boolean envvar for turning Mailchimp on/off and have it be off by default, so everyone is not Mailchimping all the time if they don't mean to?

But even that might still result in people leaving it turned on and accidentally sending loads of emails into our Mailchimp account.

I think you can actually create "test API keys" that allow you to call the Mailchimp API but don't actually send any emails, perhaps the devdata API key could be one of those.